setup.py

```diff
@@ -5,7 +5,7 @@
 from setuptools import setup, find_packages
 
 NAME = "NeutronImaging"
-META_PATH = os.path.join("neutronimaging", "__init__.py")
+META_PATH = os.path.join("../ToFImaging/jupyter_notebooks/code/neutronimaging", "__init__.py")
```

Comment:
This path change seems to be causing issue with the build server.
Please revert the change so that we can get the build to pass.